doc/source/serve/advanced-guides/direct-ingress.md

```diff
@@ -1,122 +0,0 @@
-(serve-direct-ingress)=
-# Experimental gRPC Ingress
-
-In the 2.1, Serve provides an alpha version of [gRPC](https://grpc.io/) ingress.
-
-With RPC protocol, You will get:
-
-* Standardized inference request/response schema during client and serve.
-* High performant endpoint than HTTP protocol.
-
-In this section, you will learn how to
-
-* use Serve's built-in gRPC schema to receive client traffic
-* bring your own gRPC schema into your Serve application
-
-## Use Serve's Schema
-
-Serve provides a simple gRPC schema to machine learning inference workload. It is designed to be kept simple, and you are encouraged to adapt it for your own need.
-```
-message PredictRequest {
-  map<string, bytes> input = 2;
-}
-
-message PredictResponse {
-  bytes prediction = 1;
-}
-
-service PredictAPIsService {
-  rpc Predict(PredictRequest) returns (PredictResponse);
-}
-```
-
-Take a look at the following code samples for using `DefaultgRPCDriver` in Ray Serve.
-
-To implement the Serve, your class needs to inherit `ray.serve.drivers.DefaultgRPCDriver`.
-```{literalinclude} ../doc_code//direct_ingress.py
-:start-after: __begin_server__
-:end-before: __end_server__
-:language: python
-```
-
-Client:
-You can use Serve's built-in gRPC client to send query to the model.
-
-```{literalinclude} ../doc_code//direct_ingress.py
```

Comment:
Please remove the `doc_code` files as well!

Reply:
good call!